pandas/tests/indexes/multi/test_formats.py

```diff
@@ -118,6 +118,28 @@ def test_repr(self, idx):
            names=['first', 'second'], length=6)"""
             assert result == expected
 
+        # display.max_seq_items == n
```

Comment:
prob better to make a new test as easier to see / run

Reply:
Got it. Moved this to it's own test case